nservicebus/upgrades/gateway-3to3.1.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Gateway Upgrade Version 3 to 3.1
+summary: How to upgrade the Gateway from version 3 to 3.1
+component: Gateway
+reviewed: 2019-12-17
+related:
+ - nservicebus/upgrades/6to7
+isUpgradeGuide: true
+---
+
+The gateway supports a new storage configuration API:
+
+snippet: 3to31StorageAPI
+
+This replaces:
+
+snippet: 2to3EnableGatewayAfter
```

Comment:
the two snippets would be exactly the same at this point (unless using inmem storage), not sure I get the value of the upgrade guide?

Reply:
well, not the same, different API but removed the upgrade guide as discussed